packages/portfolio-contract/test/portfolio.flows.test.ts

```diff
@@ -485,33 +501,41 @@ test('open portfolio with Aave position', async t => {
   t.log(log.map(msg => msg._method).join(', '));
   t.like(log, [
     { _method: 'monitorTransfers' },
-    { _method: 'localTransfer', amounts: { AaveAccount: { value: 50n } } },
-    { _method: 'transfer', address: { chainId: 'axelar-3' } },
-    { _method: 'localTransfer', amounts: { Aave: { value: 300n } } },
+    {
+      _method: 'localTransfer',
+      amounts: {
+        Deposit: { value: 300n },
+        GmpFee: { value: 150n },
+      },
+    },
     { _method: 'transfer', address: { chainId: 'noble-5' } },
+    { _method: 'transfer', address: { chainId: 'axelar-6' } },
     { _method: 'depositForBurn' },
-    { _method: 'localTransfer', amounts: { AaveGmp: { value: 100n } } },
-    { _method: 'transfer', address: { chainId: 'axelar-3' } },
+    { _method: 'transfer', address: { chainId: 'axelar-6' } },
     { _method: 'exit', _cap: 'seat' },
   ]);
   t.snapshot(log, 'call log'); // see snapshot for remaining arg details
   t.is(passStyleOf(actual.invitationMakers), 'remotable');
   await documentStorageSchema(t, storage, docOpts);
 });
 
+test.skip('reject missing fee before committing anything', t => {
```

Comment:
I'd prefer `test.failing` over `test.skip`, if that makes sense to you.
```suggestion
test.failing('reject missing fee before committing anything', t => {
```

Reply:
This will be cleaned up soon enough; it works in [the withdraw branch](https://github.com/Agoric/agoric-sdk/tree/dc-ymx-evm-to-noble).